src/styling.rs

```diff
@@ -0,0 +1,788 @@
+//! Style mathematical symbols in Unicode.
+
+use std::fmt::{self, Write};
+use std::iter::FusedIterator;
+
+/// The version of [Unicode](https://www.unicode.org/) that this version of the
```

Comment:
```suggestion
/// The version of [Unicode](https://www.unicode.org/versions/Unicode16.0.0/core-spec/) that this version of the
```

This may be a more useful link.

Reply:
Though that would then need updating with each version change as well. Not sure if this is really much of a point though.

Reply:
Well, the tuple below will have to be updated as well anyway. But alternatively you can replace `Unicode16.0.0` with `latest`.

Reply:
That could also cause issues as `latest` might not match the version here :sweat_smile: